### ISSUE TITLE
Sending voice memo

### ISSUE THREAD

**Issue by LeonCheong** [Iridium-IO/SiriShortcuts#10]
I used the voice memo app on ios. For some short memo, I am able to send it to PC. For some longer memo, such as one memo that is 11Mb in size, the copy on PC is corrupted.

I tried to zip the memo file and sent it as a zip file. It work.  Looks like this is not a file size issue.
I also tried to share the same memo via G drive. The audio file is ok when I dl it from GDrive. Looks like it is not a bug in iOS share

I tried SSH version v9.2.2.0p1-Beta, https://github.com/PowerShell/Win32-OpenSSH/releases/tag/v9.2.2.0p1-Beta.  Same outcome

**Comment by Iridium-IO** [comment]
Yeah I don't know why that happens, it's 100% Apple's fault. I used to be able to send a few gigabytes worth of video at once, then after an update it stopped working. Shortcuts is extremely buggy so there's not much I can do sadly :(

**Comment by LeonCheong** [comment]
Regardless, this is still an excellent piece of work. Thank you so much!
